lib/components/dropdown.vue

```diff
@@ -43,15 +43,6 @@
     </div>
 </template>
 
-<style scoped>
-    .dropdown-item:focus,
-    .dropdown-item:hover,
-    .dropdown-header:focus {
-        background-color: #eaeaea;
```

Comment:
Yep. Also darken would be done without directly giving bg color.
```cs
.dropdown-item:focus:not(.active),
.dropdown-item:hover:not(.active),
.dropdown-header:focus:not(.active)
.dropdown-header:hover:not(.active) {
    /* @See https://github.com/twbs/bootstrap/issues/23329 */
    box-shadow: inset 0px 0px 400px 110px rgba(0, 0, 0, .09);
}

.dropdown-item:active,
.dropdown-header:active {
    box-shadow: initial;
}
```
> but it might affect other things.

Hmm we could add prefix for bv class names too. You think that would be problematic?